azext_iot/_help.py

```diff
@@ -400,58 +481,90 @@
 
 helps['iot edge'] = """
     type: group
-    short-summary: Deploy and manage IoT solutions on the Edge.
+    short-summary: Manage IoT solutions on the Edge.
```

Comment:
this summary feels a bit plain to me...Something copied from this page and maybe with a link is more useful: https://docs.microsoft.com/en-us/azure/iot-edge/how-iot-edge-works

Reply:
good idea